arbitrator/src/tree_planner.cpp

```diff
@@ -87,20 +87,26 @@ namespace arbitrator
                 // Compute cost for each child and store in open list
                 for (auto child = children.begin(); child != children.end(); child++)
                 {
-                    
+
                     if (child->maneuvers.empty())
                     {
                         RCLCPP_WARN_STREAM(rclcpp::get_logger("arbitrator"), "Child was empty for id: " << std::string(child->maneuver_plan_id));
-                        continue;   
+                        continue;
                     }
 
                     temp_open_list.push_back(std::make_pair(*child, cost_function_->compute_cost_per_unit_distance(*child)));
                 }
             }
-            
+
             open_list_to_evaluate = temp_open_list;
         }
 
+        if (final_open_list.empty())
+        {
+            RCLCPP_ERROR_STREAM(rclcpp::get_logger("arbitrator"), "None of the strategic plugins generated any valid plans! Please check if any is turned and returning valid maneuvers, shutting down arbitrator...");
+            throw std::runtime_error("None of the strategic plugins generated any valid plans! Please check if any is turned and returning valid maneuvers, shutting down arbitrator...");
```

Comment:
This error message is slightly misleading. The `TreePlanner` is not shutting down the arbitrator node here. It only reports that there is an error.

Reply:
Removed the log